contracts/PetPark.sol

```diff
@@ -2,6 +2,72 @@
 pragma solidity ^0.8.0;
 
 
+
 contract PetPark {
 
+    event Added(uint animalType, uint count);
+    event Borrowed(uint animalType);
+    event Returned(uint animalType);
+
+    address public owner;
+    mapping(uint => uint) shelter;
+    mapping(address=>uint) borrowedAnimals;
+    mapping(address=>mapping(string=>uint)) borrowers;
+
+    // need to make sure that the borrower from the contract always send the same age and gender
+
+    constructor() {
+        owner = msg.sender;
+    }
+
+    function add(uint animalType, uint count) public {
+        require(msg.sender == owner, "Not an owner");
```

Comment:
Better to use a modifier to check for owner, it provides better code readability